framework/console/controllers/HelpController.php

```diff
@@ -258,7 +257,7 @@ protected function getModuleCommands($module)
                 if ($this->validateControllerClass($controllerClass)) {
                     $dir = ltrim(pathinfo($relativePath, PATHINFO_DIRNAME), '\\/');
 
-                    $command = Inflector::camel2id(substr(basename($file), 0, -14), '-', true);
+                    $command = $this->camel2id(substr(basename($file), 0, -14));
```

Comment:
Any explanation why this was changed? Original issue was only about action names.